_posts/this-week-in-solid/next.md

```diff
@@ -1,29 +1,40 @@
 ---
```

Comment:
I don't quite understand the edits in this file, if it is just copy and then edit, or if it is an accident.

Reply:
People can make PRs to this file for inclusion in the next edition, then once we actually publish it (this PR), its content is moved to a week-specific edition. This way, we can use the same contribution link everywhere, rather than having to update it to next week's edition every week.

Reply:
TWiS would definitely need separate PRs for review to be meaningful (cfr. my comment on the main thread). Should be a branch.

Reply:
They are, but the PRs are made to the `staging` branch and then reviewed by the Creators. The editor review only comes in when deploying to solidproject.org (i.e. this PR). We can change the process if the editors want to review those individually.